notebook/notebookapp.py

```diff
@@ -187,7 +187,12 @@ def init_settings(self, ipython_app, kernel_manager, contents_manager,
             },
             version_hash=version_hash,
             ignore_minified_js=ipython_app.ignore_minified_js,
-            
+
+            # rate limits
+            iopub_msg_rate_limit=ipython_app.iopub_msg_rate_limit,
+            iopub_data_rate_limit=ipython_app.iopub_data_rate_limit,
+            limit_window=ipython_app.limit_window,
```

Comment:
sorry for being pedantic, but perhaps call this `rate_limit_window`, since in the config context it's not clear what would be limited.